engine/crates/engine-v2/src/sources/graphql/mod.rs

```diff
@@ -118,6 +128,11 @@ impl<'ctx> GraphqlExecutor<'ctx> {
             &mut serde_json::Deserializer::from_slice(&bytes),
         );
 
+        subgraph_request_span.record_gql_response(GqlResponseAttributes {
+            has_errors: self.response_part.has_errors(),
+            operation_type: Some(operation_type),
```

Comment:
it's weird to have `operation_type` here, it's not something coming from the response and was already set beforehands

Reply:
yep this is probably a left over, removing.